shopgate.php

```diff
@@ -171,6 +173,8 @@ public function install()
             'header',
             'adminOrder',
             'updateOrderStatus',
+            'updateQuantity',
```

Comment:
@Quetzacoalt91 Am I correct, that the passed $module variable contains an instance of class object "ShopGate" (extends PaymentModule) or is this object a plain "PaymentModule" one?